test/metabase/data_apps/api/data_app_test.clj

```diff
@@ -183,22 +187,27 @@
               (is (false? (:retracted (first releases))))))))
 
       (testing "Create second definition and release"
-        (let [config2 {:actions []
-                       :parameters []
-                       :pages [{:name "Updated page"}]}
+        (let [config2           {:actions    []
+                                 :parameters []
+                                 :pages      [{:name "Updated page"}]}
               second-definition (mt/user-http-request :crowberto :put 200 (data-apps.tu/data-app-url app-id "/definition")
-                                                      {:config config2})
-              release2-response (mt/user-http-request :crowberto :post 200 (data-apps.tu/data-app-url app-id "/release"))]
-          (is (=? {:retracted false} release2-response))
-
-          (testing "Second release is active, first release is retracted"
-            (let [releases (t2/select :model/DataAppRelease :app_id app-id {:order-by [[:id :asc]]})]
-              (is (= 2 (count releases)))
-              (is (true? (:retracted (first releases))))
-              (is (false? (:retracted (second releases))))))
-
+                                                      {:config config2})]
+          (mt/user-http-request :crowberto :post 200 (data-apps.tu/data-app-url app-id "/release"))
           (testing "Only the latest release is returned when fetching the app"
             (let [app-with-release (mt/user-http-request :crowberto :get 200 (data-apps.tu/data-app-url app-id))]
-              (is (=? {:release {:retracted false
-                                 :app_definition_id (:id second-definition)}
+              (is (=? {:release    {:app_definition_id (:id second-definition)}
                        :definition {:revision_number 2}} app-with-release)))))))))
+
+(deftest disallow-updating-status-in-update-app-api-test
```

Comment:
Since this test includes testing the other API we should rename it.

Something short and generic like `status-update-api-test` maybe?